### PR TITLE
fix: Fixed issue with guage crashing when TickPrefix and TickSuffix don't exist

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: pkger/clone_resource.go [hunk 1]
```diff
@@ -248,6 +248,8 @@ func convertCellView(cell influxdb.Cell) chart {
 	case influxdb.SingleStatViewProperties:
 		setCommon(chartKindSingleStat, p.ViewColors, p.DecimalPlaces, p.Queries)
 		setNoteFixes(p.Note, p.ShowNoteWhenEmpty, p.Prefix, p.Suffix)
+		ch.TickPrefix = p.TickPrefix
+		ch.TickSuffix = p.TickSuffix
```
Comment (glinton) [inline pkger/clone_resource.go]: `ch` made me think these were channels at first

Comment (zoesteinkamp) [reply pkger/clone_resource.go]: Should we change the name? or you think its fine to stay?

Comment (jsteenb2) [reply pkger/clone_resource.go]: nah dont chagne the name please 😬, I hear you @glinton, but we all know chans don't have methods or members ;-).... or even useful zero values lol

**Comment by hoorayimhelping** [comment]
@zoesteinkamp a friendly tip for: try to describe what specific issue is being fixed in the PR title or description

It's not uncommon at all to look back over PRs from a while ago. If we look at this in a few months, we probably will have to do a bit of thinking to remember what was happening. If we look at this in two years, when perhaps some of the people in the review are working in other areas, we'll have even less context.

not a lot of context:
`Fixed issue with guage`

much more context:
`Fixed issue with guage crashing when TickPrefix and TickSuffix don't exist`

it takes just a little bit of context into what the the underlying issue is to make things a lot easier for people without much context into this :)

**Pull request merged**